### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by abdotaker608**

### CODE REVIEW

File: packages/home/frontend/components/commercetools-ui/organisms/account/account-atoms/alter-form.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const AlterForm: FC<AlterFormProps> = ({ page }) => {
   return (
     <div className="mt-36 border-t border-t-neutral-400 pt-40 md:mt-56 md:pt-56 lg:mt-84 lg:pt-84">
       <div className="m-auto grid max-w-[480px] px-16">
-        <Typography fontFamily="libre" fontSize={16} className="text-primary-black md:text-20 lg:text-24" as="h3">
+        <Typography className="text-16 text-primary-black md:text-20 lg:text-24" as="h3">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/account-atoms/alter-form.tsx:41]: Fonts need to be specified at least in the classNames

File: packages/home/frontend/components/commercetools-ui/organisms/account/index.tsx [hunk 2]
```diff
@@ -150,7 +150,11 @@ const AccountDetails: React.FC<AccountDetailsProps> = ({
                 {isLoading ? (
                   <Skeleton />
                 ) : (
-                  <Typography className={tabButtonLabelClassNames(tab)} fontSize={16} medium={tab.href === hash}>
+                  <Typography
+                    className={`${tabButtonLabelClassNames(tab)} text-16 ${
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/index.tsx:154]: Wouldn't it be better if we added it inside the `tabButtonLabelClassNames` ?

File: packages/home/frontend/components/commercetools-ui/organisms/account/index.tsx [hunk 3]
```diff
@@ -177,7 +181,7 @@ const AccountDetails: React.FC<AccountDetailsProps> = ({
           <div className="mt-20 px-16">
             {contentTitle && (
               <div className="block md:hidden">
-                <Typography as="h2" fontFamily="libre" className="text-18 text-primary-black">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/index.tsx:180]: Same for Libre font family

File: packages/home/frontend/components/commercetools-ui/organisms/account/sections/addresses/index.tsx [hunk 4]
```diff
@@ -56,19 +56,15 @@ const Addresses = () => {
   return (
     <div className="mt-20 px-16 md:px-24 lg:mt-40 lg:px-44">
       <div className="hidden pb-12 md:block md:pb-20 2xl:pb-36">
-        <Typography as="h2" fontFamily="libre" className="text-22 text-primary-black lg:text-24">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/sections/addresses/index.tsx:59]: Same for font family

File: packages/home/frontend/components/commercetools-ui/organisms/account/sections/orders/index.tsx [hunk 5]
```diff
@@ -124,7 +123,11 @@ const Orders = () => {
                       onClick={() => setSelectedTab(tab.slug)}
                       className="w-fit cursor-pointer whitespace-nowrap"
                     >
-                      <Typography fontSize={14} medium={tab.slug === selectedTab} className={tabTextClassNames(tab)}>
+                      <Typography
+                        className={`${tabTextClassNames(tab)} text-14 ${
+                          tab.slug === selectedTab ? 'font-medium' : 'font-normal'
+                        }`}
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/sections/orders/index.tsx:129]: There's already a `tabTextClassNames` just add the other class names there

File: packages/home/frontend/components/commercetools-ui/organisms/account/sections/orders/index.tsx [hunk 6]
```diff
@@ -143,7 +146,11 @@ const Orders = () => {
                     onClick={() => setSelectedTab(tab.slug)}
                     className="cursor-pointer whitespace-nowrap pr-36"
                   >
-                    <Typography fontSize={16} medium={tab.slug === selectedTab} className={tabTextClassNames(tab)}>
+                    <Typography
+                      className={`${tabTextClassNames(tab)} ${
+                        tab.slug === selectedTab ? 'font-medium' : 'font-normal'
+                      }`}
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/sections/orders/index.tsx:152]: Same

File: packages/home/frontend/components/commercetools-ui/organisms/account/sections/payment-methods/index.tsx [hunk 7]
```diff
@@ -40,15 +40,15 @@ const PaymentMethods = () => {
   return (
     <div className="px-16 md:mt-24 md:px-24 lg:mt-40 lg:px-44">
       <div className="hidden md:block">
-        <Typography as="h2" fontFamily="libre" fontSize={18} className="text-primary-black md:text-22 lg:text-24">
+        <Typography as="h2" className="text-18 text-primary-black md:text-22 lg:text-24">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/sections/payment-methods/index.tsx:43]: Font Family

File: packages/home/frontend/components/commercetools-ui/organisms/account/sections/payment-methods/payment-add/index.tsx [hunk 8]
```diff
@@ -19,7 +19,7 @@ const PaymentAdd = () => {
   return (
     <div className="ml-0 mt-20 lg:ml-44 lg:mt-40">
       <div className="mt-24 px-16 md:mt-0 md:px-24 lg:px-0">
-        <Typography as="h2" fontFamily="libre" fontSize={16} className="text-primary-black md:text-22 lg:text-24">
+        <Typography as="h2" className="text-primary-black md:text-22 lg:text-24">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/account/sections/payment-methods/payment-add/index.tsx:22]: Font Family

File: packages/home/frontend/components/commercetools-ui/organisms/authentication/register/register-form.tsx [hunk 9]
```diff
@@ -93,7 +93,7 @@ const RegisterForm: FC<Props> = ({ termsOfUseLink }) => {
 
   return (
     <>
-      <Typography as="h3" fontFamily="libre" className="mb-16 text-16 md:mb-24 md:text-20 lg:text-24">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/authentication/register/register-form.tsx:96]: Font family issues

File: packages/home/frontend/components/commercetools-ui/organisms/authentication/reset-password/reset-password-form.tsx [hunk 10]
```diff
@@ -74,16 +74,12 @@ const ResetPasswordForm: FC<ResetPasswordProps> = ({ token, accountLink, signInL
   };
   return (
     <>
-      <Typography as="h3" fontFamily="libre" className="mb-16 text-16 md:mb-24 md:text-20 lg:text-24">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/authentication/reset-password/reset-password-form.tsx:77]: Font Family

File: packages/home/frontend/components/commercetools-ui/organisms/header/header-navigation/header-navigation-desktop/menu-dropdown/header-menu-tile.tsx [hunk 11]
```diff
@@ -23,14 +23,12 @@ const HeaderDropdownTile: FC<Props> = ({ tileContent }) => {
         alt={tileContent?.tileImage.title ?? ''}
       />
       <div className="absolute left-1/2 top-[42%] h-35 w-full -translate-x-1/2 -translate-y-1/2">
-        <Typography fontSize={28} fontFamily="libre" as="h3" align="center" className={tileHeaderTextClassName}>
+        <Typography as="h3" className={`${tileHeaderTextClassName} text-center text-28`}>
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/header/header-navigation/header-navigation-desktop/menu-dropdown/header-menu-tile.tsx:26]: Font Family and className

File: packages/home/frontend/components/commercetools-ui/organisms/product/product-slider/index.tsx [hunk 12]
```diff
@@ -68,12 +68,7 @@ const ProductSlider: FC<ProductSliderProps> = ({
   return (
     <Wrapper background="neutral-200" clearDefaultStyles={clearDefaultWrapperStyles}>
       <div>
-        <Typography
-          className={`mb-12 ${titleClassName} ${classNames.title ?? ''}`}
-          fontSize={20}
-          as="h3"
-          fontFamily="libre"
-        >
+        <Typography className={`mb-12 text-20 ${titleClassName} ${classNames.title ?? ''}`} as="h3">
```
Comment (KenwayTheGreat) [inline packages/home/frontend/components/commercetools-ui/organisms/product/product-slider/index.tsx:71]: Shouldn't we use `classnames` ?

**Review by KenwayTheGreat — APPROVED** [review]

File: packages/home/frontend/components/commercetools-ui/organisms/account/sections/payment-methods/index.tsx (hunk 7 above)
Comment (abdotaker608) [reply packages/home/frontend/components/commercetools-ui/organisms/account/sections/payment-methods/index.tsx:43]: We have the `as=h2` here and by default headers have libre font

File: packages/home/frontend/components/commercetools-ui/organisms/product/product-slider/index.tsx (hunk 12 above)
Comment (abdotaker608) [reply packages/home/frontend/components/commercetools-ui/organisms/product/product-slider/index.tsx:71]: We don't have it in the goodstore

**Review by xcintas — APPROVED** [review]

**Pull request merged**